securedrop/tests/functional/journalist_navigation_steps.py

```diff
@@ -139,8 +147,10 @@ def _journalist_uses_delete_collection_button_confirmation(self):
 
         # After deletion the button will redirect us. Let's ensure we still
         # see the delete collection button.
-        assert self.driver.find_element_by_id(
-            'delete-collection-link').is_displayed()
+        def delete_collection_link_displayed():
+            assert self.driver.find_element_by_id(
+                'delete-collection-link').is_displayed()
+        self.wait_for(delete_collection_link_displayed, timeout=60)
```

Comment:
@dachary I see timeout only in this wait_for and not the other wait_for function calls. Shouldn't it be there in others as well?

Reply:
The default time of 10sec should be fine for the other wait_for. I set it to a generous 60 sec because this is where the race happened a few times today on my laptop.

Reply:
Okay. Makes sense then I guess.